dev/enc28j60/enc28j60.c

```diff
@@ -128,17 +127,38 @@
 PROCESS(enc_watchdog_process, "Enc28j60 watchdog");
 
 static uint8_t initialized = 0;
+static uint8_t bank = ERXTX_BANK;
 static uint8_t enc_mac_addr[6];
 static int received_packets = 0;
 static int sent_packets = 0;
 
 /*---------------------------------------------------------------------------*/
 static uint8_t
+is_mac_mii_reg(uint8_t reg)
+{
+  /* MAC or MII register (otherwise, ETH register)? */
+  switch(bank) {
+  case ERXTX_BANK:
+  case EPKTCNT_BANK:
+  default:
+    return 0;
```

Comment:
Probably a personal preference, but I would put these three cases to the bottom of the switch statement as it improves readability if the default case is the last one.